telemetry/anonymous.schema.json

```diff
@@ -59,7 +59,10 @@
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
     },
     "meta": {
```

Comment:
meta is not part of the common ping format http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/common-ping.html